code/modules/chemistry/tools/food_and_drink.dm

```diff
@@ -1136,7 +1136,8 @@
 		var/mob/living/carbon/human/H = user
 		var/list/choices = list()
 
-		if ((H.sims && H.sims.getValue("Bladder") <= 65) || (!H.sims && H.urine >= 2))
+		var/bladder = H.sims?.getValue("Bladder")
+		if (((bladder != null) && (bladder <= 65)) || ((bladder == null) && (H.urine >= 2)))
```

Comment:
```suggestion
		if ((bladder && (bladder <= 65)) || (!bladder && (H.urine >= 2)))
```

Reply:
The null checks are necessary because 0 is a possible non-null value for getValue("Bladder"), for when you *really* need to go and are seconds away from pissing yourself.

Reply:
Oh lol whoops. My bad :sheltersweat:

Reply:
Fixed in latest commit.